MediaBrowser.Controller/SyncPlay/PlaybackRequests/PlayGroupRequest.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using MediaBrowser.Controller.Session;
+using MediaBrowser.Model.SyncPlay;
+
+namespace MediaBrowser.Controller.SyncPlay
+{
+    /// <summary>
+    /// Class PlayGroupRequest.
+    /// </summary>
+    public class PlayGroupRequest : IGroupPlaybackRequest
+    {
+        /// <summary>
+        /// Gets the playing queue.
+        /// </summary>
+        /// <value>The playing queue.</value>
+        public List<Guid> PlayingQueue { get; } = new List<Guid>();
```

Comment:
```suggestion
        public IReadOnlyList<Guid> PlayingQueue { get; } 
```

Additionally, set it from constructor